app/lib/actions/katello/applicability/repository/regenerate.rb

```diff
@@ -12,8 +12,9 @@ class Regenerate < Actions::EntryAction
 
           def run
             ::Katello::Repository.find(input[:repo_id]).content_facets.each do |facet|
```

Comment:
I just realized, i think this will skip host's that are attached to a Content View instance of the repo.  We probably need to look up content_facets associated with any clone of repo_id  as well

Reply:
and to make clear, i think this was an issue on the initial applicabiliyt pr

Reply:
Oh yep I can see that happening. I'll fix